src/main/java/Interactors/IOController.java

```diff
@@ -11,10 +12,10 @@ public class IOController {
     private UseCaseInteractor interactor;
     private State currentState;
 
-    public IOController(){
+    public IOController(File file){
         this.output = "Welcome to Monopoly! Do you want to ";
         this.input = 0;
-        this.interactor = new UseCaseInteractor();
+        this.interactor = new UseCaseInteractor(new TextFileTranslator(file));
```

Comment:
I am gonna assume that the use case interactor is the one that handles the cases for if a file should be read or not. I think for functionality wise this works for now, but I think a better choice for how the cases should be implemented is through the IO controller, which we can handle in a later PR.